components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/header.adoc

```diff
@@ -8,19 +8,23 @@ In Starknet, a block is a list of transactions, and a block header that contains
 
 [%autowidth]
 |===
-| Name | Type | Description | Implemented
-
-| `parent_block_hash` | `FieldElement` | The hash of this block's parent | &#10003;
-|`block_number` | `Integer` | The number (height) of this block | &#10003;
-| `global_state_root` | `FieldElement` | The state xref:../Network_Architecture/starknet-state.adoc#state_commitment[commitment] after this block | &#10003;
-|`sequencer_address` | `FieldElement` | The Starknet address of the sequencer who created this block | &#10003;
-| `block_timestamp` | `Timestamp` | The time the sequencer created this block before executing transactions | &#10003;
-|`transaction_count` | `Integer` | The number of transactions in a block | &#10003;
-| `transaction_commitment` | `FieldElement` | A commitment to the transactions included in the block | &#10003;
-|`event_count` | `Integer` | The number of events | &#10003;
-| `event_commitment` | `FieldElement` | A commitment to the events produced in this block | &#10003;
-| `protocol_version` | `Integer` | The version of the Starknet protocol used when creating this block |
-| `extra data` | `FieldElement` | Extraneous data that might be useful for running transactions |
+| Name | Type | Description
+
+| `parent_block_hash` | `FieldElement` | The hash of this block's parent
+|`block_number` | `Integer` | The number (height) of this block
+| `global_state_root` | `FieldElement` | The state xref:../Network_Architecture/starknet-state.adoc#state_commitment[commitment] after this block
+|`sequencer_address` | `ContractAddress` | The Starknet address of the sequencer who created this block
+| `block_timestamp` | `Timestamp` | The time the sequencer created this block before executing transactions
```

Comment:
Does this mean:
1. The amount of time prior to beginning to execute the transaction in this block 
2. The time, in hours, since the Unix epoch in UTC, at which the sequencer began building the block
3. The time, in hours, since the Unix epoch in UTC, at which the sequencer began executing the block
4. Something else?